src/EditorComponent.jsx

```diff
@@ -7,12 +7,49 @@ import { exampleSetup } from 'prosemirror-example-setup';
 import { keymap } from 'prosemirror-keymap';
 import { InputRule, inputRules } from 'prosemirror-inputrules';
 import { toggleMark } from 'prosemirror-commands';
+import AppConfig from './AppConfig';
 
 import secondsToHhmmss from './utils/secondsToHhmmsss';
 
+function floorOrZero(n) {
+    return Number.isNaN(n) ? 0 : Math.floor(n);
+}
+
+const ImageNodeSpec = {
```

Comment:
All the extra line space is unnecessary. It makes the reviewer scroll more and spend more time jumping from one line to another.